src/gatling/scala/SingleLeaseMultipleChecks.scala

```diff
@@ -0,0 +1,46 @@
+import com.typesafe.config.{Config, ConfigFactory}
+import com.warrenstrange.googleauth.GoogleAuthenticator
+import io.gatling.core.Predef._
+import io.gatling.http.Predef._
+
+import scala.concurrent.duration._
+
+class SingleLeaseMultipleChecks extends Simulation {
+  val config: Config = ConfigFactory.load()
+
+  val authenticator: GoogleAuthenticator = new GoogleAuthenticator()
+
+  val requestJwt = exec(
+    http("Lease token")
+      .post("/lease")
+      .formParam("microservice", config.getString("service.name"))
+      .formParam("oneTimePassword", authenticator.getTotpPassword(config.getString("service.pass")))
```

Comment:
I think this password will be used every time. We need to provide these passwords using a feeder.

Reply:
Some OTP are the same, however, they are generated with 30 seconds expiration, and tests can last longer than that.